llvm/include/llvm/Demangle/ItaniumDemangle.h

```diff
@@ -445,6 +445,14 @@ class EnableIfAttr : public Node {
   }
 };
 
+#ifdef _MSC_VER
+// Workaround for MSVC++ bug (Version 2017, 15.8.9) - w/o this forward
+// declaration, the friend declaration in ObjCProtoName below has no effect
+// and leads to compilation error when ObjCProtoName::Protocol private field
+// is accessed in PointerType::printLeft.
+class PointerType;
```

Comment:
Please, amend https://reviews.llvm.org/D81905 with this workaround.

Reply:
Yes, will do.

Reply:
For the record - done.